src/gsi/read_satwnd.f90

```diff
@@ -212,7 +214,7 @@ subroutine read_satwnd(nread,ndata,nodata,infile,obstype,lunout,gstime,twind,sis
   
   real(r_double),dimension(13):: hdrdat
   real(r_double),dimension(4):: obsdat
-  real(r_double),dimension(2) :: hdrdat_test
+  real(r_double),dimension(2) :: hdrdat_test,hdrdat_005099
```

Comment:
Please check if hdrdat_005099 is used or not. I did not see it in new code.

Reply:
Yes it is used in line 1299 & line 1300.
                 call ufbint(lunin,hdrdat_005099,2,1,iret, 'GNAPS PCCF');
                 qifn=hdrdat_005099(2);